akka-actor-typed/src/main/scala/akka/actor/typed/internal/receptionist/LocalReceptionist.scala

```diff
@@ -55,8 +55,8 @@ private[akka] object LocalReceptionist extends ReceptionistBehaviorProvider {
     def watchWith(ctx: ActorContext[Any], target: ActorRef[_], msg: InternalCommand): Unit =
       ctx.spawnAnonymous[Nothing](Behaviors.setup[Nothing] { innerCtx ⇒
         innerCtx.watch(target)
-        Behaviors.immutable[Nothing]((_, _) ⇒ Behaviors.same)
-          .onSignal {
+        Behaviors.receive[Nothing]((_, _) ⇒ Behaviors.same)
+          .receiveSignal {
```

Comment:
isn't that simply `Behaviors.receiveSignal` `

Reply:
yep! thanks